### DISCUSSION TITLE
Sandbox features still needed

### DISCUSSION THREAD

**Discussion by Michael-Lalime** [ioos/Cloud-Sandbox#80] [Ideas]
@Michael-Lalime @mwengren @KatherinePowell-NOAA @ZacharyWills 
It looks like CO-OPS will be running the Coastal Ocean Reannalysis (CORA) 40-year Pacific reannalysis project entirely in the sandbox rather than on an NOAA R&DHPC system. The project new technical lead, John Ratcliffe, will start on May 14th and the project will start running in June. I think we need to have a discussion about the sandbox features they will need before they start running. I'm thinking about attaching an S3 bucket and/or some way to move data in and out of the sandbox without copying through the bastion server, post processing tools (Nebari?), connections to NODD, and anything else we need to put in place before they start so we can prioritize them. 

@ZacharyWills we need to include realistic timelines for when we can get these and other unidentified features and enhancements completed for tracking progress and reporting up the chain. Everything is getting critical now and the project has become much more visible and is under increased scrutiny so we all need to be aware of what that means. I would like each feature to have it's own Github issue and all communications, draft versions, and discussions to take place in those issues to help increase the visibility of the work being done. 

Does this seem reasonable to everyone? Does it seem like a good idea to use Github for this discussion and issue tracking? Also, I'm not sure if Mykel has a Github account but maybe we can work on that so he can be involved too. 

@mwengren does Tiffany have an account on Github and what do you think of using Github for this discussion and issue tracking?

**Comment by mwengren** [comment]
@Michael-Lalime Oooh, that is fantastic news!  There will definitely be a lot to talk about to make sure we understand the requirements and can build technical capabilities accordingly. 

I'm in favor of using GitHub for discussion, requirements gathering, etc, as long as there is no sensitive/internal information that will be necessary to discuss.  Perhaps we should meet with the CORA group to make sure they're on the same page as far as that goes.  

I'd recommend we create a project board for this repo (& any others that are related to the Sandbox effort - model specific forks etc that Patrick has created that are deployed to the head node to run models), so we can better track issue backlog, work progress, etc.

I don't have a good idea for how to decide whether to use GH Discussions or regular Issues.  Initially (even though this topic is under Discussions) I'd say we should default to using Issues unless there's a compelling reason to use Discussions instead.

**Comment by Michael-Lalime** [comment]
Creating a project board sounds like a good idea Micah. I'm not sure if GH
Discussions was the correct place or not. I had never used it before and
thought it might be good to try for the initial discussion and am perfectly
happy using issues. Once concern that was raised was about the open access
nature to Github and what information we might reveal. Maybe the project
board can be more private and exclusive to the group and collaborators. We
can discuss more tomorrow.
